.github/workflows/fetch-otp-data.yml

```diff
@@ -0,0 +1,74 @@
+name: 'Fetch and process data to build OTP graph'
+
+on:
+  workflow_call:
+  workflow_dispatch:
+    inputs:
+      with_elevation:
+          description: 'Include elevation file in OTP data source'
+          default: false
+          type: boolean
+      local_download:
+          description: 'Output otp_dir artifact, default uploads to S3 bucket'
+          default: false
+          type: boolean
```

Comment:
I could imagine a scenario where we might want to fetch and process latest feeds for local testing/manual fixes, so I added these inputs in just in case